generators/app/templates/javascript/common/_package.json

```diff
@@ -7,19 +7,21 @@
     "node": ">=10"
   },
```

Comment:
we are not using package.json from common right ? I guess you forgot to delete this file.